packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -727,44 +957,343 @@ export class ModularChangeFamily
 		);
 	}
 
+	// This performs a first pass on all fields which have both new and base changes.
+	// TODO: Can we also handle additional passes in this method?
+	private rebaseIntersectingFields(
+		crossFieldTable: RebaseTable,
+		rebasedNodes: ChangeAtomIdMap<NodeChangeset>,
+		genId: IdAllocator,
+		constraintState: ConstraintState,
+		metadata: RebaseRevisionMetadata,
+	): FieldChangeMap {
+		const change = crossFieldTable.newChange;
+		const baseChange = crossFieldTable.baseChange;
+		const rebasedFields = this.rebaseFieldMap(
+			change.fieldChanges,
+			baseChange.fieldChanges,
+			undefined,
+			genId,
+			crossFieldTable,
+			metadata,
+		);
+
+		// This loop processes all fields which have both base and new changes.
+		// Note that the call to `rebaseNodeChanges` can add entries to `crossFieldTable.nodeIdPairs`.
+		for (const [newId, baseId, _attachState] of crossFieldTable.nodeIdPairs) {
+			const rebasedNode = this.rebaseNodeChange(
+				newId,
+				baseId,
+				genId,
+				crossFieldTable,
+				metadata,
+				constraintState,
+			);
+
+			if (rebasedNode !== undefined) {
+				setInChangeAtomIdMap(rebasedNodes, newId, rebasedNode);
+			}
+		}
+
+		return rebasedFields;
+	}
+
+	// This processes fields which have no base changes but have been invalidated by another field.
```

Comment:
What's a scenario where this happens?

Reply:
I didn't realize this until you asked, but we currently don't use this. Earlier representations of move marks could require updating the other endpoint (which might be in a field with no base changes) when rebasing a move.

Reply:
Do you want to keep it in the PR anyway and remove it later so we have it in the history? If not I'd just remove it. What do you think?